embulk-output-jdbc/src/main/java/org/embulk/output/jdbc/AbstractJdbcOutputPlugin.java

```diff
@@ -693,10 +694,11 @@ protected TableIdentifier buildIntermediateTableId(final JdbcOutputConnection co
                 {
                     intermTables = new ArrayList<>();
                     if (task.getMode().tempTablePerTask()) {
-                        String namePrefix = generateIntermediateTableNamePrefix(task.getActualTable().getTableName(), con, 3,
+                        int suffixLength = Math.max(1, String.valueOf(taskCount - 1).length());
```

Comment:
Sorry for nitpicking…, but this `suffixLength` calculation part could be in the `build…` method by passing `taskCount`? That would be more modular and would have better test coverage.

Reply:
@dmikurube This is because `suffixLength` is passed as an argument to the `genereateIntermediateTableNamePrefix` method. If testing this calculation, I'll have to create another method like `calculateSuffixLength`, is that ok?

Reply:
Or, is it better to create a separate implementation of `genereateIntermediateTableNamePrefix` that takes `taskCount` as an argument, and embed the `suffixLength` calculation in each of its methods and the `build...` method?

Reply:
See. https://github.com/embulk/embulk-output-jdbc/pull/301/commits/02eeae77008a7c4f59a7d132aed0a8030bb81b2c